schemas/com.snowplowanalytics.monitor/status_report/jsonschema/1-0-0

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a status report event emitted from an application",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitor",
+    "name": "status_report",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The ID of the status report; a UUID",
+      "type": "string",
```

Comment:
Do we need an ID for the event? (It will automatically be assigned an event ID and timestamp if it is a Snowplow event.)

Reply:
No immediate need, it's more that we can't have an empty schema for this event. But to make the case for a dedicated ID, in future we might want to report other events from inside the app and be able to assign them to a specific status report. This would act like `page_view_id` does on web.

Reply:
👍